code/modules/paperwork/photocopier.dm

```diff
@@ -332,3 +345,6 @@
 	grind_results = list(/datum/reagent/iodine = 40, /datum/reagent/iron = 10)
 	var/charges = 5
 	var/max_charges = 5
+
+
+#undef MAX_TONER
```

Comment:
no need for define

Reply:
It's a magic number used in 2 places. I want a define.

Reply:
```suggestion

```
![image](https://user-images.githubusercontent.com/13056792/72291197-289e4a00-364f-11ea-8c86-c6e41602dc7a.png)